packages/core/src/components/menu/_menu.scss

```diff
@@ -342,6 +346,15 @@ Styleguide components.menu.css.header
       }
     }
 
+    &:not(.pt-disabled) {
```

Comment:
🔥  look, how hard is it to not have to use `:not()` here and just override properties when disabled? 

this is hard to grok because these are actually the _default_ un-interacted styles (i think?) but the nesting makes me think they're for a modifier, when it's actually the inverse.